bndtools.core/src/bndtools/wizards/bndfile/NewBndFileWizardPage.java

```diff
@@ -57,8 +57,14 @@ protected boolean validatePage() {
         IResource container = ResourcesPlugin.getWorkspace().getRoot().findMember(containerPath);
         IProject project = container.getProject();
 
-        if (Project.BNDFILE.equalsIgnoreCase(fileName)) {
-            error = Messages.NewBndFileWizardPage_errorReservedFilename;
+        try {
+            if (project.hasNature(BndtoolsConstants.NATURE_ID)) {
+                if (Project.BNDFILE.equalsIgnoreCase(fileName)) {
+                    error = Messages.NewBndFileWizardPage_errorReservedFilename;
```

Comment:
When I try to add a `bnd.bnd` to a non-Bndtools project today I get an error say "This file name is reserved"